igarss_2023/igarss_mtedgecumbe_ts_analysis.ipynb

```diff
@@ -181,6 +252,14 @@
     "plt.tight_layout()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "4b17efa6-3371-425a-bcb9-49cafece2bb6",
+   "metadata": {},
+   "source": [
+    "This section walked you through requesting and downloading one interferogram, but time series InSAR routinely make use of hundreds of interferograms (Grapenthin et. al., 2022 used XX!). Unfortunately we don't have the time to request and download this many interferograms during this workshop, so instead we've pre-generated and prepped some InSAR for you to work with. Think of it as your very own InSAR cooking show!"
```

Comment:
```suggestion
    "We have now gone through the process of requesting and downloading a single interferogram, but time series InSAR workflows routinely make use of hundreds of interferograms (Grapenthin et al., 2022 used XX!). We don't have time to request and download this many interferograms during this workshop, so instead we've pre-generated and prepped some InSAR products for you to work with. Think of it as your very own InSAR cooking show!"
```

Reply:
A general comment:
The phrase "time series InSAR" is used frequently, and that always reads a bit strangely to me. I would tend to use "InSAR time series [analysis]", but I've also never really done it, so it's purely semantic to me, not based in common usage in the field. Is it common to use the construction with the InSAR following time series instead of the other way around?

Reply:
@hjkristenson I think "time series InSAR" is used slightly more often than "InSAR time series" when describing the technique, but both are used. To me, "InSAR time series" sounds like you're describing a specific data stack (e.g. "take a look at this InSAR time series"), and "time series InSAR" sounds like you're referencing the analysis technique (e.g., "I performed a time series InSAR analysis").